### PR TITLE
Loa 266 set up navigation controller

### PR DESCRIPTION
**Pull request by Kostanix**

### CODE REVIEW

File: app/src/main/java/de/hdmstuttgart/thelaendofadventure/FullscreenActivity.kt [hunk 1]
```diff
@@ -1,199 +1,36 @@
...
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupActionBarWithNavController
 import de.hdmstuttgart.the_laend_of_adventure.R
 import de.hdmstuttgart.the_laend_of_adventure.databinding.ActivityFullscreenBinding
-import de.hdmstuttgart.thelaendofadventure.data.Tracking
-import de.hdmstuttgart.thelaendofadventure.ui.fragments.UserCreationFragment
-import kotlinx.coroutines.launch
 
 /**
```
Comment (ayal104) [inline app/src/main/java/de/hdmstuttgart/thelaendofadventure/FullscreenActivity.kt:12]: Please update the javadoc comment.

File: app/src/main/java/de/hdmstuttgart/thelaendofadventure/FullscreenActivity.kt [hunk 2]
```diff
@@ -1,199 +1,36 @@
...
-        isFullscreen = true
 
-        fullscreenContentControls = binding.fullscreenContentControls
+        Log.d("Main", "ContainerCreated!")
 
-        // Upon interacting with UI controls, delay any scheduled hide()
-        // operations to prevent the jarring behavior of controls going away
-        // while interacting with the UI.
-        binding.dummyButton.setOnTouchListener(delayHideTouchListener)
+        val navHostFragment = supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
```
Comment (ayal104) [inline app/src/main/java/de/hdmstuttgart/thelaendofadventure/FullscreenActivity.kt:29]: Can "findFragmentById" be replaced with binding?

**Review by ayal104 — APPROVED** [review]
LGTM!

File: app/src/main/java/de/hdmstuttgart/thelaendofadventure/FullscreenActivity.kt (hunk 2 above)
Comment (Kostanix) [reply app/src/main/java/de/hdmstuttgart/thelaendofadventure/FullscreenActivity.kt]: I tried, its not possible

**Pull request merged**